packages/uniforms/__tests__/filterDOMProps.ts

```diff
@@ -11,11 +11,18 @@ describe('joinName', () => {
   });
 
   it('removes registered props', () => {
+    // @ts-ignore: Do not register its type not to pollute it.
     filterDOMProps.register('__special__');
 
     expect(filterDOMProps({ __special__: true })).toEqual({});
   });
 
+  it('ignores double registers', () => {
+    const { length } = filterDOMProps.registered;
```

Comment:
Isn't it simpler? 
```suggestion
    const length = filterDOMProps.registered.length;
```

Reply:
Ah, I'm also never sure about the readability of that... The generated code is the same, though. Let's leave it for now - it's only in tests.